components/style/traversal.rs

```diff
@@ -520,6 +522,8 @@ where
         }
     }
 
+    notify_paint_worklet(context, data);
```

Comment:
Are you sure you want this outside of the `if compute_self` block? Do we really always need to paint it even if the style is the same?